getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Understanding Qubits and Quantum Gates\n",
+    "\n",
+    "## What are Qubits ?\n",
+    "\n",
+    "The bit is the fundamental concept of classical computation and classical information. Quantum computation and quantum information are built upon an analogous concept, the quantum bit, or _qubit_ for short.\n",
+    "\n",
+    "Physical implementations of Qubits vary depending on the type of technology used for the Quantum Computer. For this text, we'll consider Qubits as abstract mathematical objects. \n",
+    "\n",
+    "Just as a classical bit has a state – either 0 or 1 – a qubit also has a state. Two possible states for a qubit are the states |0⟩ and |1⟩ - as represented in [_Bra-Ket or Dirac notation_](https://en.wikipedia.org/wiki/Bra%E2%80%93ket_notation). The state of the qubit is a unit vector in a two-dimensional complex vector space and are represented as : \n",
```

Comment:
Two possible states -> Two possible basis states;
and are represented -> and the two basis states are represented

Reply:
Updated